pyproject.toml

```diff
@@ -45,6 +45,12 @@
     name = "Contributor-facing changes"
     showcontent = true
 
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [
+  "setuptools >= 45",
+  "setuptools_scm[toml] >= 6.2",
```

Comment:
I just realized that this PR is missing the bit allowing Git archive installs to work:
```suggestion
  # Essentials
  "setuptools >= 45",

  # Plugins
  "setuptools_scm[toml] >= 6.2",
  "setuptools_scm_git_archive>=1.1",
```
(it's basically a part of the usual boilerplate)

Another part of this would be adding `.git_archival.txt` with the following content:
```txt
ref-names: $Format:%D$
```

And `.gitattributes` containing:
```txt
# Force LF line endings for text files
* text=auto eol=lf

# Needed for setuptools-scm-git-archive
.git_archival.txt  export-subst
```

Reply:
What's the use case here?  I'm not sure this should/needs to be done in this PR.

Reply:
It must be included. Otherwise, things like `pip install https://github.com/ansible/ansible-navigator/archive/refs/tags/2.0.0.tar.gz` will produce broken versions.